pages/index.tsx

```diff
@@ -2,15 +2,18 @@ import type { NextPage } from 'next';
 import Head from 'next/head';
 
 import styles from '../styles/Home.module.css';
-import { flatMocks } from '../lib/mock/mock';
+import useSWR from 'swr';
 import { FlatList } from '../components/FlatList';
 import type { FlatType } from '../lib/types/entities';
 
 type HomePageProps = {
   preloadedFlatList: FlatType[];
 };
-
-const Home: NextPage<HomePageProps> = ({ preloadedFlatList }) => {
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+const Home: NextPage<HomePageProps> = () => {
+  const { data, error } = useSWR('/api/getFlats', fetcher);
```

Comment:
we add one empty row before and after constant or constantS. If constant is placed on the top of function - only after constant or constantS

Reply:
something like in Airbnb https://github.com/airbnb/javascript/tree/master/react